### ISSUE TITLE
[Form]: FormGroup `titleText` not announced by screenreader on entering that particular region

### ISSUE THREAD

**Issue by rama1990br** [SAP/ui5-webcomponents-react#6480]
### Describe the bug

`titleText` is not read out by the screenreader when entering the Form Group region and focusing on the first FormItem element within the group (or the last FormItem element on re-entering the group). This is required to be read out so that the blind users get the required context while navigating through the form.

One way that this can be achieved is by enclosing the form group in a div and adding `role` and `aria-label` attributes on the `div` and setting the `aria-label` to the `titleText`.

### Isolated Example

https://stackblitz.com/edit/github-m52y88?file=src%2FApp.tsx

### Reproduction steps

1. Open JAWS 2024 (or an earlier version) and have it running
2. Open the above isolated example and tab to the first input 'Name' within 'Personal Data' form group. Observe that 'Personal Data' is not read out by the screenreader. 
3. Continue tabbing through till 'Company Name' within 'Company Data' form group, and then press shift+tab to re-enter the previous form group by focusing on 'Home address' checkbox. Observe that 'Personal Data' is not announced by the screenreader in this instance as well.

### Expected Behaviour

The form group `titleText` should be read out by the screenreader on entering and re-entering the form group region.

### UI5 Web Components for React Version

1.28.4

### UI5 Web Components Version

1.24.0

### Browser

Chrome, Edge, Firefox, Safari

### Organization

SAP SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @rama1990br 

the example you provided is using the v2 form, but in your description you're referring to v1. Please let us know which version you're using, so we can continue to process this issue.
